scripts/data_files/driver_jsons/driver_opaque_schema.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "prefix": {
+      "type": "string"
```

Comment:
Well, I autogenerated the schema based on a valid JSON file, is there a way I can autogenerate this  with acceptable range or so ?

Reply:
I have no idea, I presume this would depend on the autogenerator. But anyway, why does the autogenerator matter at this point? I expect the schemas to be maintained manually. The schema comes from the specification, not from inference from some examples.